press_kit.md

```diff
@@ -0,0 +1,174 @@
+
+## **INDEX:**
+
+**[1. AsyncAPI](#_heading=h.40s9q7rdmzv1)**
+
+[1.1. Open governance model](#_heading=h.m14n0k7zs2qm) 
+
+[1.2. AsyncAPI, in numbers](#_heading=h.f8xlcwy3vi6) 
+
+**[2. History of the project](#_heading=h.5yg403h4t8gi)**
+
+**[3. Values of AsyncAPI](#_heading=h.4me7screlap)**
+
+**[4. Brands and companies using AsyncAPI](#_heading=h.xxo7nwvlczto)**
+
+**[5. Funding](#_heading=h.bx8fkbp32vqc)**
+
+**[6. Contact details](#_heading=h.7oacawrz9cjj)**
+
+**[7. Frequently asked questions](#_heading=h.9m5tif8t2hnq)**
+
+## **1. ASYNCAPI**
+The initiative is an open source specification that has developed an asynchronous messaging language within the event-driven architecture (EDA). This tool enables the standardization of machine-to-machine communication. **The main aim of the project is to make asynchronous APIs as successful and mature as REST APIs.** This programming system covers a hitherto unexplored field. There is no direct competition as it is inserted in a new and emerging market.
+
+The project is based on transparency, teamwork and passion, cooperating with other free software initiatives. Also, AsyncAPI can't be understood without the talent and the contributions of its community. It has been crucial for getting to this point.
+
+Inspired by the logic of OpenAPI, in less than five years it has managed to carve out a recognized position for itself in the world of APIs. This project is seeking to improve the current state of event-driven architectures (EDA).
+
+And, although it was born out of the need to fill a gap in the standardization of asynchronous APIs, the differential value of the initiative is that it has created its sense of purpose and has grown without a profit motive.
+
+One of the main challenges is to make the development experience super seamless, from idea to production. Users should be able to create their first API within minutes without prior knowledge of AsyncAPI, the communication protocol, or anything else that's not business logic.
+
+Right now, the strategy of the project is to continue improving the specification, implementing resources that make it more complete and solid. This has led to the partnership with Postman (December 2020), a leading collaborative platform for API development and a reference within this sector all around the world. This partnering allows it to boost the development of more and better tools to help engineers create and maintain Asynchronous APIs while using their favorite programming languages and frameworks.
+
+The next step has been to host the project on a neutral foundation to ensure the long-term sustainability and success of the initiative. AsyncAPI has been hosted by the Linux Foundation since March of 2021.
+
+However, none of the other goals are possible without the support of a large community. AsyncAPI is and must remain a community-driven initiative.
+
+### **1.1. Open governance model**

```

Comment:
I'd remove the numbers and the bold format from all the headings.

Reply:
Is this because it is usually put this way in .md documents or for some other reason? Thanks :)

Reply:
Maintaining the numbers is always a pain. Tomorrow you add a new section in the middle and you'll have to change half the numbers in the document. Also, Github automatically generates "links" to headings. With numbers, the link would look something like github.com/Barbanio/Press-Kit/blob/README.md#1-1-open-governance-model instead of github.com/Barbanio/Press-Kit/blob/README.md#open-governance-model.

No strong opinion, just saying it can become a pain and they don't add any relevant information. One can always click on the link and it will scroll to the exact position. I understand this is done in books or printed publications because you can't click on a link but this is an interactive medium.

Up to you :)

Reply:
Makes sense, thanks :)

Reply:
It then makes sense to remove the numbers from the index as well, right?

Reply:
Yes, Markdown will introduce the numbers for you :)

Reply:
Thanks :)